examples/Toast.close.jsx

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { useToaster, Button, ProgressRadial } from '@itwin/itwinui-react';
+
+export default () => {
+  const toaster = useToaster();
+
+  const displayProcessToast = () => {
+    const { close } = toaster.informational(
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'flex-end',
+        }}
+      >
+        <ProgressRadial
+          size={'small'}
+          indeterminate
+          style={{ marginRight: '8px' }}
+        />
+        Your process is running...
+      </div>,
```

Comment:
Could we just use flex here?

```jsx
<Flex>
  <ProgressRadial size='small'/>
  Your process is running…
</Flex>
```

Reply:
I have made the change.

Reply:
nit: The `ProgressRadial` can also be simplified as per my suggestion above